### PR TITLE
adding exploring prague recommendation

### PR DESCRIPTION
**Pull request by Cheukting**

### CODE REVIEW

**Comment by Cheukting** [comment]
Ummm I didn't see the preview, where is it?

File: src/content/pages/explore.mdx [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+title: More Tips on Exploring Prague
+subtitle: In EuroPython there are many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surraounding area?
```
Comment (cybit) [inline src/content/pages/explore.mdx:3]:
```suggestion
subtitle: In EuroPython there are many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surrounding area?
```

File: src/content/pages/explore.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+import venueImage from "./images/prague.jpg";
+
+# More Tips on Exploring Prague
+
+<Image src={venueImage} width="1200" alt="" />
+
+<br></br>
+
+In EuroPython there are many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surraounding area?
```
Comment (cybit) [inline src/content/pages/explore.mdx:16]:
```suggestion
At EuroPython many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surrounding area?
```

File: src/content/pages/explore.mdx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+Here are some tips and things to do provided by our volunteers, hope it helps you and your companions will have a great time in Prague:
```
Comment (cybit) [inline src/content/pages/explore.mdx:18]:
```suggestion
Here are some tips and things to do provided by our volunteers, hope it helps you and your companions to have a great time in Prague:
```

File: src/content/pages/explore.mdx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+## Traveling around
+
+### Taking Public transport
```
Comment (cybit) [inline src/content/pages/explore.mdx:22]:
```suggestion
### Public transport
```

File: src/content/pages/explore.mdx [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+In Prague the recommended way to get around is public transport:
+
+- There’s some info [here](https://ep2024.europython.eu/where#getting-around-in-prague)
+- You can use your normal card to travel around by train within Prague
+- Usually fast and reliable
+- After midnight no metro, but there are buses
```
Comment (cybit) [inline src/content/pages/explore.mdx:29]:
```suggestion
- After midnight the metro stops. - But there are buses
```

File: src/content/pages/explore.mdx [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+Some people may consider hiring a car but:
+
+- In Prague it’s unnecessary, it’s hard to find parking and it’s expensive
+- You can hire a car at the airport or anywhere (like you normally do in other cities)
+- If you want to visit other towns, you can use trains as well (they’re mostly cheap)
+
+### Be prepared for the weather
+
+If you want to explor the city, you will be exposed to the weather. So here are teh answers to the questions: What’s the weather like? What to carry with you?
```
Comment (cybit) [inline src/content/pages/explore.mdx:42]:
```suggestion
If you want to explore the city, you will be exposed to the weather. So here some answers to frequent questions like: What’s the weather like? What to carry with you?
```

File: src/content/pages/explore.mdx [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+- Usually 30°C+, so carry water
+- There are no fountains nor public toilets, so carry more water
```
Comment (cybit) [inline src/content/pages/explore.mdx:45]:
```suggestion
- There are no water fountains nor public toilets, so carry more water
```

File: src/content/pages/explore.mdx [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+- Use Sunscreen
+- It rarely rains, but when it does it’s shower rain - it’s the best to hide somewhere until it stops and it usually stops after half an hour or so.
```
Comment (cybit) [inline src/content/pages/explore.mdx:47]:
```suggestion
- It rarely rains, but when it does it’s shower rain - it’s best to just hide somewhere until it stops (it usually stops after half an hour or so).
```

File: src/content/pages/explore.mdx [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+## What to do while in Prague?
+
+There are plenty of thing you can do in Prague, depending on what is your preference, you can find things that interests you and your group.
```
Comment (cybit) [inline src/content/pages/explore.mdx:51]:
```suggestion
There are plenty of thing you can do in Prague. Depending on your preference, you can find things that interests you and your group.
```

File: src/content/pages/explore.mdx [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+### For culture lovers
+
+- Summer cinema
+- Theater festival Za dveřmi 2024
+
+### For art lovers
+
+- DOX
+- Soví mlýny
```
Comment (cybit) [inline src/content/pages/explore.mdx:61]: What is this in english?

File: src/content/pages/explore.mdx [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+- National museum
+- Jazz dock for live jazz on a boat
+- Žižkov area for bars and pubs
+- House of the Black Madonna: cafe shop in a building that’s built in cubist style (there was cubist architecture movement in prague and it’s quite famous and unique)
+
+### For nature lovers
+
+- Petřín hill: Eiffel tower (in Prague), nicce view of the city
```
Comment (cybit) [inline src/content/pages/explore.mdx:69]:
```suggestion
- Petřín hill: Eiffel tower (in Prague), nice view of the city
```

File: src/content/pages/explore.mdx [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+- Vyšehrad: nice fortress next to the venue, park, you can sit and chill
```
Comment (cybit) [inline src/content/pages/explore.mdx:70]:
```suggestion
- Vyšehrad: nice fortress next to the venue. It also features a park where you can sit and chill
```

File: src/content/pages/explore.mdx [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+- Letná: viewpoint
+- Riegrovy Sady: park with nice view, there’s a rose garden there that most people don’t know about - you should check it out
```
Comment (cybit) [inline src/content/pages/explore.mdx:72]:
```suggestion
- Riegrovy Sady: park with nice view. There’s a rose garden most people don’t know about - you should check it out
```

File: src/content/pages/explore.mdx [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+- Divoká šárka: small forest a bit further away, you can swim in Džbán (artificial lake)
+- Kunratický les: forest
+- Hostivář: area for walk, you can swim there as well
+
+*Use mapy.cz - there are routes there (red, blue and green ones) - the best maps for hiking in general*
```
Comment (cybit) [inline src/content/pages/explore.mdx:77]:
```suggestion
*Use [mapy.cz](https://mapy.cz/) - there are routes there (red, blue and green ones) - the best maps for hiking in general*
```

File: src/content/pages/explore.mdx [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+
+## Where to eat?
+
+Prague is a food city! There are many places you will find nice food, from traditional Czechia food to Vegan friendly food.
```
Comment (cybit) [inline src/content/pages/explore.mdx:81]:
```suggestion
Prague is a food city! There are many places to find nice food, from traditional Czechia food to Vegan friendly food.
```

File: src/content/pages/explore.mdx [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+- 👎 Pražská šunka on the street: tourist trap
+- 👍 Try Nakládaný hermelín (pickled camembert)
+- 👍 Smažený sýr (fried cheese)
+- 👍 Svíčková (meat with vegetable sauce and dumplings)
+- 👍 Guláš (goulash)
+
+
+### For veggie/ vegans
+
+We know some of us are veggie/ vegan, don't worrie you will find places that serve you veggie/ vegan food:
```
Comment (cybit) [inline src/content/pages/explore.mdx:97]:
```suggestion
We know some of us are veggie / vegan. Don't worry, you will find places that serve you veggie / vegan food:
```

File: src/content/pages/explore.mdx [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+
+- Shromazdiste: typical czech food in vegan versions, great food and great beer
+- Eateria: vegan junk food, best burger
+- Pastva: open during day only, european and healthy food
+- Dhaba beas: cheap buffet, indian and european food, it’s everywhere
+- Share Sweet & Espresso Bar: coffee shop with best vegan cakes
+
+According to HappyCow there are 89 vegan and vegetarian places to eat, so it’s not an issue. Typical Czech restaurants offer mostly only fried cheese and french fries, sometimes pickled camembert, but czech cuisine is mostly meat-heavy. If hungry during the night, go to any kebab place and you can order falafel wrap with veggies. You can also go to Burrito Loco - it’s mexican food chain, they offer vegan mana burrito and burrito with grilled veggies.
```
Comment (cybit) [inline src/content/pages/explore.mdx:105]:
```suggestion
According to HappyCow there are 89 vegan and vegetarian places to eat. So it should not be an issue to find something. Typical Czech restaurants mostly only offer fried cheese and french fries, sometimes pickled camembert.  Czech cuisine is mostly meat-heavy. 
Hungry during the night? Go to any kebab place and you can order falafel wrap with veggies. You can also go to Burrito Loco - it’s a mexican food chain and they offer vegan mana burrito and burrito with grilled veggies.
```

File: src/content/pages/explore.mdx [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+
+On top of the above:
+
+### All day social places
+
+- Automat Matuška
+
+### Recommendation for music lovers
+
+There are festivals around you can check them:
```
Comment (cybit) [inline src/content/pages/explore.mdx:128]:
```suggestion
There are multiple festivals going on you might want to check out:
```

File: src/content/pages/explore.mdx [hunk 19]
```diff
@@ -0,0 +1,143 @@
...
+
+- Jazz dock for jazz on boat
```
Comment (cybit) [inline src/content/pages/explore.mdx:130]:
```suggestion
- Jazz dock for jazz on a boat
```

File: src/content/pages/explore.mdx (hunk 18 above)
Comment (cybit) [reply src/content/pages/explore.mdx:128]: Do we have links to all the following festivals?

Has anybody checked if they are on between July 7th and 14th?

File: src/content/pages/explore.mdx [hunk 20]
```diff
@@ -0,0 +1,143 @@
...
+- Lucerna
+- Prague Open Air festival
+- Prague Rooftop festival
+- Letní scéna Musea Kampa 2024 festival music + theater
+- Jazz festival Mladí ladí jazz 2024
+
+[You can find more events here](https://goout.net/en/prague/events/leznyvlkk/?after=2024-07-08%2000%3A00&before=2024-07-14%2023%3A59)
+
+### For party animals and clubing enthusiates
```
Comment (cybit) [inline src/content/pages/explore.mdx:139]:
```suggestion
### For party animals and clubbing enthusiasts
```

File: src/content/pages/explore.mdx (hunk 10 above)
Comment (Cheukting) [reply src/content/pages/explore.mdx:61]: @clytaemnestra do you know?

**Comment by Cheukting** [comment]
Thanks @cybit

File: src/content/pages/explore.mdx (hunk 10 above)
Comment (clytaemnestra) [reply src/content/pages/explore.mdx:61]: > Museum Owls' mills (Soví mlýny)

File: src/content/pages/explore.mdx [hunk 21]
```diff
@@ -0,0 +1,144 @@
+---
+title: More Tips on Exploring Prague
+subtitle: In EuroPython there are many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surrounding area?
```
Comment (clytaemnestra) [inline src/content/pages/explore.mdx:3]: I'd make paste everything into ChatGPT and ask it to fix the grammar only - I see lots of typos, grammar mistakes etc.

**Comment by clytaemnestra** [comment]
More ideas cross my mind:
* Žižkov tower - viewpoint (I recommend at night) + baby sculptures on it
* the Elevator of Death - the elevator moving all the time up and down without stopping
* for partying - street Dlouhá třída
* Botanical garden Troja for nature lovers
* sightseeing boat trip 
* hipster area with good coffee - Vinohrady, for example La Bohème café 
* area to grab drinks in the evening - Náplavka riverbanks, lots of nice bars and bars on boats

File: src/content/pages/explore.mdx [hunk 22]
```diff
@@ -0,0 +1,144 @@
...
+- Letní scéna Musea Kampa 2024 festival music + theater
+- Jazz festival Mladí ladí jazz 2024
+
+[You can find more events here](https://goout.net/en/prague/events/leznyvlkk/?after=2024-07-08%2000%3A00&before=2024-07-14%2023%3A59)
+
+### For party animals and clubbing enthusiasts
+
+- Roxy: all genres
+- Lucerna: pop, sometimes alternative
+- Cross club: punk, alternative
```
Comment (egeakman) [inline src/content/pages/explore.mdx:144]: Ran the whole thing through ChatGPT to fix grammar mistakes

```suggestion
---
title: More Tips on Exploring Prague
subtitle: At EuroPython, many attendees will come with their families. While not attending the conference, why not go and explore Prague and the surrounding area?
---

import { Image } from "astro:assets";
import venueImage from "./images/prague.jpg";

# More Tips on Exploring Prague

<Image src={venueImage} width="1200" alt="" />

<br></br>

At EuroPython, many attendees will come with their families. While not attending the conference, why not go and explore Prague and the surrounding area?

Here are some tips and things to do provided by our volunteers. We hope they help you and your companions have a great time in Prague:

## Traveling around

### Public transport

In Prague, the recommended way to get around is public transport:

- There’s some info [here](https://ep2024.europython.eu/where#getting-around-in-prague)
- You can use your normal card to travel around by train within Prague
- It is usually fast and reliable
- After midnight, the metro stops, but there are buses
- Mostly safe, with no no-go stations or anything similar

### Do I need to rent a car?

Some people may consider renting a car, but:

- In Prague, this is unnecessary. It’s hard to find parking, and it’s expensive
- You can rent a car at the airport or other places in the city (like you normally do in other cities)
- If you want to visit other towns, you can use trains as well (they’re mostly cheap)

### Be prepared for the weather

If you want to explore the city, you will be exposed to the weather. So here are some answers to frequent questions like: What’s the weather like? What should you carry with you?

- Usually 30°C+, so carry water
- There are no water fountains or public toilets, so carry more water
- Use sunscreen
- It rarely rains, but when it does, it’s shower rain. It’s best to just hide somewhere until it stops (it usually stops after half an hour or so).

## What to do while in Prague?

There are plenty of things you can do in Prague. Depending on your preferences, you can find things that interest you and your group.

### For culture lovers

- Summer cinema
- Theater festival Za dveřmi 2024

### For art lovers

- DOX
- Soví mlýny
- National Museum
- Jazz Dock for live jazz on a boat
- Žižkov area for bars and pubs
- House of the Black Madonna: cafe shop in a building that’s built in cubist style (there was a cubist architecture movement in Prague and it’s quite famous and unique)

### For nature lovers

- Petřín Hill: Eiffel Tower (in Prague), nice view of the city
- Vyšehrad: nice fortress next to the venue. It also features a park where you can sit and chill
- Letná: viewpoint
- Riegrovy Sady: park with a nice view. There’s a rose garden most people don’t know about - you should check it out
- Divoká Šárka: small forest a bit further away, you can swim in Džbán (an artificial lake)
- Kunratický Les: forest
- Hostivář: area for walking, you can swim there as well

*Use [mapy.cz](https://mapy.cz/) - there are routes there (red, blue, and green ones) - the best maps for hiking in general*

## Where to eat?

Prague is a food city! There are many places to find nice food, from traditional Czech food to vegan-friendly options.

### General recommendations

Some yays and nays from our local volunteers:

- 👎 Trdelnik is a tourist thing: nothing original or authentic
- 👎 Pražská šunka on the street: tourist trap
- 👍 Try Nakládaný Hermelín (pickled camembert)
- 👍 Smažený sýr (fried cheese)
- 👍 Svíčková (meat with vegetable sauce and dumplings)
- 👍 Guláš (goulash)

### For veggies/vegans

We know some of us are veggies/vegans. Don't worry, you will find places that serve you veggie/vegan food:

- Shromáždiště: typical Czech food in vegan versions, great food, and great beer
- Eateria: vegan junk food, best burger
- Pastva: open during the day only, European and healthy food
- Dhaba Beas: cheap buffet, Indian and European food, it’s everywhere
- Share Sweet & Espresso Bar: coffee shop with the best vegan cakes

According to HappyCow, there are 89 vegan and vegetarian places to eat. So it should not be an issue to find something. Typical Czech restaurants mostly only offer fried cheese and French fries, sometimes pickled camembert. Czech cuisine is mostly meat-heavy. Hungry during the night? Go to any kebab place and you can order a falafel wrap with veggies. You can also go to Burrito Loco - it’s a Mexican food chain and they offer vegan mana burrito and burrito with grilled veggies.

[Visit HappyCow vegan-vegetarian listings](https://www.happycow.net/europe/czech_republic/prague/?filters=vegan-vegetarian)

## Where to drink?

In European countries, drinking is a popular way to socialize. Here are some recommendations:

- For craft beer: Sibeeria Bar, Pivotéka, BeerGeek, Zlý časy
- Pilsner-type beer: everywhere
- For mild and not bitter Pilsner-style beer, try Únětice (it’s in a few places). They also have 8 degrees with less alcohol
- For more bitter, try Plzeň/Pilsner or Kozel, Radegast
For craft, try Matuška (APA, IPA, ALE), Zíchovec (Sour ALE, APA, IPA, ALE) or Sibeeria (Sour ALE, APA, IPA, ALE)
- For wine: Bublinka 2024 wine festival

On top of the above:

### All-day social places

- Automat Matuška

### Recommendations for music lovers

There are multiple festivals going on you might want to check out:

- Jazz Dock for jazz on a boat
- Lucerna
- Prague Open Air Festival
- Prague Rooftop Festival
- Letní scéna Musea Kampa 2024 Festival: music + theater
- Jazz Festival Mladí ladí jazz 2024

[You can find more events here](https://goout.net/en/prague/events/leznyvlkk/?after=2024-07-08%2000%3A00&before=2024-07-14%2023%3A59)

### For party animals and clubbing enthusiasts

- Roxy: all genres
- Lucerna: pop, sometimes alternative
- Cross Club: punk, alternative
```

Comment (Cheukting) [reply src/content/pages/explore.mdx:144]: I am not against chat GPT but I think we need at least a pair of human eyes to final review it. I cannot see the preview generated, @egeakman can you help? Just check if chat GPT says anything that doesn't make sense

Comment (egeakman) [reply src/content/pages/explore.mdx:144]: Hey @cheukting, of course! I'll go over it tomorrow.

File: src/content/pages/explore.mdx [hunk 23]
```diff
@@ -0,0 +1,144 @@
...
+- Summer cinema
+- Theater festival Za dveřmi 2024
+
+### For art lovers
+
+- DOX
+- Soví mlýny
+- National museum
+- Jazz dock for live jazz on a boat
+- Žižkov area for bars and pubs
```
Comment (hypha) [inline src/content/pages/explore.mdx:64]: bars and pubs are lovely, but I am not sure why they are put in the art lovers' section. Do these bars and pubs also function as galleries or host gigs?

**Review by hypha — COMMENTED** [review]
Added one minor suggestion. I think it would be helpful to add URLs to the venues mentioned. I am happy to help with this after the grammar & typos get sorted out.

File: src/content/pages/explore.mdx (hunk 23 above)
Comment (clytaemnestra) [reply src/content/pages/explore.mdx:64]: Jazz dock hosts jazz concerts every night. About žižkov area - there are some pubs and bars with live music as well, but I agree that maybe it's better to move it somewhere else.

Comment (hypha) [reply src/content/pages/explore.mdx:64]: Yeah I think the jazz dock is very relevant here (and great shout!). For žižkov, maybe mention something like "many of which will host live music".

File: src/content/pages/explore.mdx (hunk 22 above)
Comment (hypha) [reply src/content/pages/explore.mdx:144]: @Cheukting I wonder if it is easier to accept the suggestions and then I am happy to review from there. It is a bit difficult now to know what has already been suggested. 
P.S. looking forward to visiting so many of these wonderful places <3 
P.P.S: I have a pair of human eyes! ☺️

Comment (egeakman) [reply src/content/pages/explore.mdx:144]: @hypha @Cheukting Ah I completely forgot this, sorry! I've gone through it; the chatgpt suggestion looks good overall, but it misses the Czech accents time to time. Like ``í`` in ``Trdelník``. Other than this, I agree with Raquel's suggestion [below](https://github.com/EuroPython/website/pull/690#pullrequestreview-2127767339).

Comment (clytaemnestra) [reply src/content/pages/explore.mdx:144]: I don't think missing accents is an issue 🙂 If you need me to add them, feel free to commit or merge it and I'm happy to go over it and add accents, where missing 🙂

Comment (hypha) [reply src/content/pages/explore.mdx:144]: Is it OK to just commit them and then we can quickly do a final pass?

Comment (egeakman) [reply src/content/pages/explore.mdx:144]: If it's OK with @Cheukting, we can commit the suggestion.

**Comment by Cheukting** [comment]
PR updated, please final check. Would love to merge this soon

File: src/content/pages/explore.mdx [hunk 24]
```diff
@@ -0,0 +1,141 @@
...
+
+### For culture lovers
+
+- Summer cinema
+- Theater festival Za dveřmi 2024
+
+### For art lovers
+
+- DOX
+- Soví mlýny
```
Comment (cybit) [inline src/content/pages/explore.mdx:60]:
```suggestion
- Soví mlýny (Museum Owls' mills)
```

File: src/content/pages/explore.mdx [hunk 25]
```diff
@@ -0,0 +1,141 @@
...
+- National Museum
+- Jazz Dock for live jazz on a boat
+- Žižkov area for bars and pubs
+- House of the Black Madonna: cafe shop in a building that’s built in cubist style (there was a cubist architecture movement in Prague and it’s quite famous and unique)
+
+### For nature lovers
+
+- Petřín Hill: Eiffel Tower (in Prague), nice view of the city
+- Vyšehrad: nice fortress next to the venue. It also features a park where you can sit and chill
```
Comment (cybit) [inline src/content/pages/explore.mdx:69]:
```suggestion
- Vyšehrad: Fortress next to the venue. It also features a park where you can sit and chill
```
All is nice. But too much nice one after the other is no longer cute

File: src/content/pages/explore.mdx [hunk 26]
```diff
@@ -0,0 +1,141 @@
...
+- Letná: viewpoint
+- Riegrovy Sady: park with a nice view. There’s a rose garden most people don’t know about - you should check it out
```
Comment (cybit) [inline src/content/pages/explore.mdx:71]:
```suggestion
- Riegrovy Sady: Park with a view. There’s a rose garden most people don’t know about - definitely check it out
```

File: src/content/pages/explore.mdx [hunk 27]
```diff
@@ -0,0 +1,141 @@
...
+- National Museum
+- Jazz Dock for live jazz on a boat
+- Žižkov area for bars and pubs
+- House of the Black Madonna: cafe shop in a building that’s built in cubist style (there was a cubist architecture movement in Prague and it’s quite famous and unique)
+
+### For nature lovers
+
+- Petřín Hill: Eiffel Tower (in Prague), nice view of the city
+- Vyšehrad: nice fortress next to the venue. It also features a park where you can sit and chill
+- Letná: viewpoint
```
Comment (cybit) [inline src/content/pages/explore.mdx:70]:
```suggestion
- Letná: Viewpoint
```

File: src/content/pages/explore.mdx [hunk 28]
```diff
@@ -0,0 +1,141 @@
...
+- Riegrovy Sady: park with a nice view. There’s a rose garden most people don’t know about - you should check it out
+- Divoká Šárka: small forest a bit further away, you can swim in Džbán (an artificial lake)
```
Comment (cybit) [inline src/content/pages/explore.mdx:72]:
```suggestion
- Divoká Šárka: Small forest a bit further away. - Go for a swim in Džbán (an artificial lake)
```

File: src/content/pages/explore.mdx [hunk 29]
```diff
@@ -0,0 +1,141 @@
...
+- Kunratický Les: forest
```
Comment (cybit) [inline src/content/pages/explore.mdx:73]:
```suggestion
- Kunratický Les: Forest
```
What is so special about this?

File: src/content/pages/explore.mdx [hunk 30]
```diff
@@ -0,0 +1,141 @@
...
+- Hostivář: area for walking, you can swim there as well
```
Comment (cybit) [inline src/content/pages/explore.mdx:74]:
```suggestion
- Hostivář: Walking area. (You can swim there as well)
```

**Comment by cybit** [comment]
@egeakman , @clytaemnestra I will merge this as is and we open a new PR to fix stuff.

**Comment by hypha** [comment]
I'll start the new one! Desperate to add links! :muscle:

**Pull request merged**